run-tests.php

```diff
@@ -3651,11 +3650,6 @@ class RuntestsValgrind
     protected bool $version_3_8_0;
     protected string $tool;
 
-    public function getHeader(): string
```

Comment:
never invoked method

Reply:
You sure? https://github.com/php/php-src/blob/9d948c1238c70a67e4160196caf0425850d7c1ee/run-tests.php#L907